static/css/style.css

```diff
@@ -117,11 +117,25 @@
 
 .text {
     margin-bottom: 10px;
+    padding: 10px;
+    border-radius: 5px;
+    background-color: #f1f1f1;
```

Comment:
One minor suggestion: I noticed that the background color around the messages could use a slight adjustment. Could you please change the background color to #fdfafa? It would make it more appealing.